src/routes/auth/create-new-user.e2e.test.js

```diff
@@ -14,7 +14,7 @@ describe('POST /auth/create-user', () => {
 				lastName: faker.name.lastName(),
 				email: faker.internet.email(),
 				password: faker.internet.password(),
-				profilePictureUrl: faker.internet.url()
+				profilePictureUrl: faker.image.imageUrl()
```

Comment:
Nice initiative but unless for some reason, `faker.internet.url` does not work at all,  not sure changing this file is within the scope of the issue

Reply:
yes please